.github/workflows/ci.yml

```diff
@@ -331,21 +367,36 @@ jobs:
     steps:
       - uses: actions/checkout@v3
       - name: Install Rust stable
-        uses: dtolnay/rust-toolchain@master
+        uses: dtolnay/rust-toolchain@stable
         with:
           toolchain: ${{ env.rust_stable }}
           target: ${{ matrix.target }}
-      - name: Install cross
-        uses: taiki-e/install-action@cross
+
+      - name: Install cargo-nextest
+        uses: taiki-e/install-action@v2
+        with:
+          tool: cargo-nextest
+      # Install linker and libraries for i686-unknown-linux-gnu
```

Comment:
```suggestion
      # Install linker and libraries
```

Reply:
Ehhh I've removed the comment.

Feel free to open another review if you want to add it back.